src/components/legend/draw.js

```diff
@@ -609,13 +609,15 @@ function computeLegendDimensions(gd, groups, traces) {
             maxItems = Math.max(maxItems, group.length);
         });
 
-        maxHeight += opts.tracegroupgap || 5;
+        maxHeight += opts.tracegroupgap;
+        var traceGroupGap = 5;
```

Comment:
To prevent confusion, maybe we should call this variable `traceGap` instead of `traceGroupGap`.

Maybe you can change this in your next commit along with updated baselines (I expect at least one baseline will change)!

Thanks again @nathanjenx

Reply:
No problemo, will get that sorted now!